src/types/enviroment.d.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2022 Joshua Schmitt
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+declare global {
+  namespace NodeJS {
+    interface ProcessEnv {
+      DB_HOST: string;
+      DB_PORT: string;
+      DB_USER: string;
+      DB_PASSWORD: string;
+      DISCORD_TOKEN: string;
+      DISCORD_CLIENT_ID: string;
```

Comment:
You could use Discord Types here. By importing with ``import { Snowflake } from "discord.js";`` you could you Snowflake as a type, this way we can preserve *.ts config file types!

Reply:
Ok, im looking at this!